apps/x509.c

```diff
@@ -489,8 +489,12 @@ int x509_main(int argc, char **argv)
         goto end;
     }
 
+    if (newcert && infile != NULL) {
+        BIO_printf(bio_err, "ignoring -in option since -newcert is set\n");
```

Comment:
There is no `-newcert` option, the option in question seems to be `-new`.  You might also update the documentation of `-new` to make clear that `-in` must not be used in this case.

Reply:
Oops, yes, I meant the `-new` option of course; corrected.

I thought that the following part of the doc of `-new`:
> (which would normally be given as B<-in filename>)
 already would make clear enough that it makes little sense

Reply:
Oops, yes, I meant the `-new` option of course; corrected.

I thought that the following part of the doc of `-new`:

> (which would normally be given as B<-in filename>)

already would make clear enough that it makes little sense to use it with `-in`.
Nevertheless I've replaced this now by:

> So the B<-in> option should not be used in this case.